manifests/prometheus/spec/alerts/uaa-5xx-errors.test.yml

```diff
@@ -1,7 +1,7 @@
 ---
 rule_files:
   # See alerts_validation_spec.rb for details of how stdin gets set:
-  - spec/alerts/fixtures/rules.yml
+  - fixtures/rules.yml
```

Comment:
What made these need to change?

Reply:
it seems to be from upgrading the promtool (see individual commits) 

> new version appears to now expect rule_files paths to be relative to the test file

Reply:
the promtool bump

Reply:
(see the commit comment)

if you think about it it makes more sense for these to be relative to the file rather than the cwd of the caller.

Reply:
:+1: